src/Persistence.LinnApps/Repositories/ReceiptPrefSupDiffRepository.cs

```diff
@@ -0,0 +1,34 @@
+﻿namespace Linn.Purchasing.Persistence.LinnApps.Repositories
+{
+    using System;
+    using System.Linq;
+    using System.Linq.Expressions;
+
+    using Linn.Common.Persistence;
+    using Linn.Purchasing.Domain.LinnApps.Reports.Models;
+
+    public class ReceiptPrefSupDiffRepository : IQueryRepository<ReceiptPrefSupDiff>
+    {
+        private readonly ServiceDbContext serviceDbContext;
+
+        public ReceiptPrefSupDiffRepository(ServiceDbContext serviceDbContext)
+        {
+            this.serviceDbContext = serviceDbContext;
+        }
+
+        public ReceiptPrefSupDiff FindBy(Expression<Func<ReceiptPrefSupDiff, bool>> expression)
+        {
+            throw new NotImplementedException();
+        }
+
+        public IQueryable<ReceiptPrefSupDiff> FilterBy(Expression<Func<ReceiptPrefSupDiff, bool>> expression)
+        {
+            return this.serviceDbContext.ReceiptPrefsupDiffs.Where(expression);
```

Comment:
I think since there aren't any includes or custom code this repository file could go and could just do the sort of new EntityFrameworkQueryRepository in the IOC PersistenceExtensions class thing. Probably doesn't matter that it's there but it might save you making an extra class in future https://github.com/linn/common/blob/master/src/Linn.Common.Persistence.EntityFramework/EntityFrameworkQueryRepository.cs